ntbs-service/Models/NotificationBannerModel.cs

```diff
@@ -113,9 +113,11 @@ public NotificationBannerModel(Notification notification, bool showPadlock = fal
             RedirectPath = RouteHelper.GetNotificationPath(notification.NotificationId, NotificationSubPaths.Overview);
         }
 
-        private static string CalculateOutcome(ICollection<TreatmentEvent> treatmentEvents)
+        private static string CalculateOutcome(ICollection<TreatmentEvent> treatmentEvents, bool isPostMortemWithCorrectEvents)
         {
-            return treatmentEvents.GetMostRecentTreatmentEvent()
+            return (isPostMortemWithCorrectEvents
+                    ? treatmentEvents.Single(te => te.TreatmentEventTypeIsOutcome)

```

Comment:
In a couple of places I do this, where I filter on outcome. As we know isPostMortemWithCorrectEvents has passed, we know that there is 1 outcome event, and it is death. I don't think this is immediately clear reading it now, maybe I should comment in these places

Reply:
I think it's fairly obvious here, because the variable you're switching on is `isPostMortemWithCorrectEvents` so it seems natural that when that's `true` we can assume something about the structure of the treatment events. However, if you're worried about it then it can't hurt to add a short comment.